projects/canopy/src/styles/grid.scss

```diff
@@ -116,7 +116,7 @@ $columns: 12;
 }
 
 @for $i from 1 through $columns {
-  $width: percentage($i / $columns);
+  $width: percentage($i / 12);
```

Comment:
is there a reason to loose the variable and replace with '12' ?

Reply:
Oh well spotted, no! I will create a PR for the fix to go into `next` 👍

Reply:
@aaronBery fix here: https://github.com/Legal-and-General/canopy/pull/697